openreview/venue/recruitment.py

```diff
@@ -87,25 +87,38 @@ def invite_committee(self,
 
         print('sending recruitment invitations')
         for index, email in enumerate(tqdm(invitees, desc='send_invitations')):
+            profile_emails = []
+            profile = None
+            is_profile_id = email.startswith('~')
+            invalid_profile_id = False
+            if is_profile_id:
+                try:
+                    profile = tools.get_profile(self.client, email)
+                except openreview.OpenReviewException as e:
+                    error_string = repr(e)
+                    if 'ValidationError' in error_string:
+                        invalid_profile_id = True
+                    else:
+                        if error_string not in recruitment_status['errors']:
```

Comment:
in which case we get this error? like an upstream error or something like that?

Reply:
I am not sure, we haven't gotten any error here yet, but I was wondering if we should still catch errors other than ValidationError in case we get any

Reply:
yes, we should catch them.